<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelConfigItem</class>
 <widget class="QWidget" name="ChannelConfigItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="channelText">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="state">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Disconnected</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="connectBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/connect.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="logBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>View log</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/log.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editBtn">
        <property name="toolTip">
         <string>Edit</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/edit.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteBtn">
        <property name="toolTip">
         <string>Delete</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/error.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectBtn</sender>
   <signal>clicked()</signal>
   <receiver>ChannelConfigItem</receiver>
   <slot>connectToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logBtn</sender>
   <signal>clicked()</signal>
   <receiver>ChannelConfigItem</receiver>
   <slot>showLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editBtn</sender>
   <signal>clicked()</signal>
   <receiver>ChannelConfigItem</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBtn</sender>
   <signal>clicked()</signal>
   <receiver>ChannelConfigItem</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showLog()</slot>
  <slot>edit()</slot>
  <slot>remove()</slot>
  <slot>connectToggle()</slot>
 </slots>
</ui>
